posthog/async_migrations/definition.py

```diff
@@ -98,6 +98,11 @@ class AsyncMigrationDefinition:
     # name of async migration this migration depends on
     depends_on: Optional[str] = None
 
+    # run before 'registering' the migration in the UI. Returns a boolean specifying if the instance should
+    # show this migration in the UI
+    def is_shown(self) -> bool:
```

Comment:
I think this is actually a bit of a misleading name - I came into this PR thinking we'd just hide this from the UI, but in reality we don't "setup" the migration - no DB entry etc.

I understand your rationale btw for the approach - it would get messy to set up the model and just hide it in the UI. Unfortunately "naming is hard" and I don't have a great suggestion. The terms "setup by default" or something come to mind.

Anyway, one actionable from this long ramble is to maybe update the comment above this function at least - I suspect it was written before the implementation took its form. Would be nice to specify that this migration will effectively be non-existent.

Also just came to mind that flipping it around might be conceptually easier to understand i.e. `is_hidden`, since the default is to show 🤷  not a blocker nor very important